spec/counter_culture_spec.rb

```diff
@@ -2285,6 +2287,17 @@ def mess_up_counts
     end
   end
 
+  it "can fix counts by scope" do
+    prefecture = Prefecture.new name: 'Tokyo'
+    prefecture.save!
+    City.create!(name: 'Sibuya', prefecture: prefecture, population: 221800)
+    City.create!(name: 'Oku Tama', prefecture: prefecture, population: 6045)
+
+    City.counter_culture_fix_counts
```

Comment:
I think the value here will already be correct? Can you manually set the `big_cities_count` value to something incorrect?

Reply:
Exactly.
Its done. https://github.com/magnusvk/counter_culture/pull/272/commits/455e81f5d70d9b0f2adffb879b4687c28f1bd437